src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -47,17 +47,11 @@
 				<NotificationsSettings :conversation="conversation" />
 			</NcAppSettingsSection>
 
-			<NcAppSettingsSection v-if="canFullModerate"
-				id="conversation-settings"
-				:title="t('spreed', 'Moderation')">
-				<ListableSettings v-if="!isNoteToSelf" :token="token" />
-				<LinkShareSettings v-if="!isNoteToSelf" ref="linkShareSettings" />
-				<ExpirationSettings :token="token" can-full-moderate />
-			</NcAppSettingsSection>
-			<NcAppSettingsSection v-else
-				id="conversation-settings"
-				:title="t('spreed', 'Setup summary')">
-				<ExpirationSettings :token="token" />
+			<NcAppSettingsSection id="conversation-settings"
+				:title="canFullModerate ? t('spreed', 'Moderation') : t('spreed', 'Setup summary')">
```

Comment:
```suggestion
				:title="canFullModerate ? t('spreed', 'Moderation') : t('spreed', 'Setup Overview')">
```

It may be understood as " setup summary = modify summary"